File: 1960-check-if-the-sentence-is-pangram/check-if-the-sentence-is-pangram.py
class Solution(object):
    def checkIfPangram(self, sentence):
        """
        :type sentence: str
        :rtype: bool
        """
        alphabets = set(['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'])

        for word in sentence:
            if word in alphabets:
                alphabets.remove(word)

        if not alphabets:
            return True
        else:
            return False

File: 1960-check-if-the-sentence-is-pangram/check-if-the-sentence-is-pangram.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    let unique_letter = [];
    for (let char of sentence){
        if (!unique_letter.includes(char)){
            unique_letter.push(char);
        }
    }

    return unique_letter.length == 26;
};